skynet/modules/stt/streaming_whisper/utils/utils.py

```diff
@@ -305,3 +305,12 @@ def get(self, time_arg_millis: int = None) -> UUID:
         uuid_int = (timestamp_ms & 0xFFFFFFFFFFFF) << 80
         uuid_int |= secrets.randbits(76)
         return UUID(int=uuid_int, version=7)
+
+
+def get_jwt(ws_headers, ws_url_param=None) -> str:
+    jwt = ws_url_param if ws_url_param else ''
```

Comment:
Shouldn't we return early if we find it in the URL?

Reply:
not really, I would like it to always pick the auth header if present. The dict key check is very fast anyway

Reply:
My comment is not about performance, but rather this looks weird in case you have both. If we prefer the header, let's check there first, else return the other. It makes the code more straightforward, no?